cmd/service.go

```diff
@@ -151,6 +177,8 @@ var serviceListCmd = &cobra.Command{
 
 func init() {
 	serviceDeleteCmd.Flags().BoolVarP(&serviceForceDeleteFlag, "force", "f", false, "Delete service without prompting")
+	serviceCreateCmd.Flags().StringVar(&plan, "plan", "", "The name of the plan of the service to be created")
+	serviceCreateCmd.Flags().StringArrayVarP(&parameters, "parameter", "p", []string{}, "Parameters of the plan where a parameter is expressed as <key>=<value")
```

Comment:
`StringSliceVarP` might be better since you can then have `-p "foo=bar,baz=boo" -p "a=b"` which will be parsed as `[]string{"foo=bar", "baz=boo", "a=b"}` while `StringArrayVarP` would result in `[]string{"foo=bar,baz=boo", "a=b"}`

Reply:
Also, would name the flag `parameters` instead of `parameter` but that's just me :)

Reply:
I agree with `parameters` instead of `parameter`. 
I am not sure about the first suggestion, because I don't have enough Cobra experience. @cdrage @kadel WDYT?